app/urls.py

```diff
@@ -21,7 +21,8 @@
     url(r'code_conduct/$', views.code_conduct, name='code_conduct'),
     url(r'legal_notice/$', views.legal_notice, name='legal_notice'),
     url(r'privacy_and_cookies/$', views.privacy_and_cookies, name='privacy_and_cookies'),
-    url(r'terms_and_conditions/$', views.terms_and_conditions, name='terms_and_conditions')
+    url(r'terms_and_conditions/$', views.terms_and_conditions, name='terms_and_conditions'),
+    url(r'^files/(?P<file>.*)$', views.protectedMedia, name="protect_media")
```

Comment:
This may fail due to this:   https://github.com/hackupc/registration/blob/62f85d769445be253214dab66827e1b7062511c9/app/urls.py#L32

Reply:
We should delete then the line you comment?

Reply:
probably, make sure it does the same though before.